packages/cli/src/commands/account/new.ts

```diff
@@ -134,9 +155,10 @@ export default class NewAccount extends BaseCommand {
     const accountAddress = toChecksumAddress(privateKeyToAddress(keys.privateKey))
 
     if (derivationPath === CELO_DERIVATION_PATH_BASE) {
+      // TODO tell them they can configure the default derivation path
```

Comment:
will handle once that functionality is done in next pr

Reply:
I'd remove that TODO tbh